payment/views.py

```diff
@@ -0,0 +1,107 @@
+from rest_framework import viewsets
+from rest_framework.response import Response
+from rest_framework import status
+import stripe
+from django.conf import settings
+from django.urls import reverse
+from django.shortcuts import get_object_or_404
+from .models import Payment
+from borrowing.models import Borrowing
+from rest_framework.permissions import IsAuthenticated
+from .serializers import PaymentSerializer
+from rest_framework.views import APIView
```

Comment:
change imports. don't use .models/.serializer, it's not good practice. Sort and add empty lines